### PR TITLE
Switches between x and o

### PR DESCRIPTION
**Pull request by ksabbak**

### CODE REVIEW

File: src/clojure_tictactoe/tictactoe.clj [hunk 1]
```diff
@@ -34,6 +34,13 @@ Examples:
       (. System exit 0)
       user-input)))
 
+(def player-atom (atom "x"))
```
Comment (BWoodfork) [inline src/clojure_tictactoe/tictactoe.clj]: I'm curious as to the reason behind using an atom here?

File: src/clojure_tictactoe/tictactoe.clj [hunk 2]
```diff
@@ -34,6 +34,13 @@ Examples:
...
+
+(defn switch-player [current-player]
```
Comment (hnlee) [inline src/clojure_tictactoe/tictactoe.clj:38]: Where's the test for this new function?

**Review by hnlee — CHANGES_REQUESTED** [review]

File: src/clojure_tictactoe/tictactoe.clj (hunk 2 above)
Comment (ksabbak) [reply src/clojure_tictactoe/tictactoe.clj:38]: ```>.>```

**Review by hnlee — APPROVED** [review]
👍

File: src/clojure_tictactoe/tictactoe.clj [hunk 3]
```diff
@@ -52,10 +59,10 @@ Examples:
 
 (defn get-player-choice
   ([]
-   (println "Which space would you like to mark, Player x?")
+   (println (str "Which space would you like to mark, Player " @player-atom "?"))
    (let [choice (parse-move-input (get-user-input))]
    (if choice
-     {choice "x"}
+     {choice @player-atom}
```
Comment (BWoodfork) [inline src/clojure_tictactoe/tictactoe.clj:64]: Do you *need* to use an atom here? I think there's an alternative to using one but could be wrong.

Atom's are *normally* frowned upon in Clojureland. There's usually a better alternative to using them. But in some cases it makes sense.

**Pull request merged**